src/gui/statusbar.cpp

```diff
@@ -86,6 +86,9 @@ StatusBar::StatusBar(QWidget *parent)
     m_upSpeedLbl->setStyleSheet(u"text-align:left;"_s);
     m_upSpeedLbl->setMinimumWidth(200);
 
+    m_externalAddressLbl = new QLabel(tr("External Address: %1").arg(0), this);
```

Comment:
Wouldn't it be better to use `IP` instead of `Address` to save some space in the UI?